tests/jobs/test_scheduler_job.py

```diff
@@ -4769,6 +4773,375 @@ def test_max_active_runs_in_a_dag_doesnt_stop_running_dagruns_in_otherdags(self,
         assert dag1_running_count == 1
         assert running_count == 11
 
+    def test_dag1_is_mutated(self, dag_maker, session):
```

Comment:
1) this seems odd - something specific to dag_maker? 2) did you mean to keep this in here?

Reply:
yeah i didn't really mean to include it. was just going to post on slack... "isn't this odd"